File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// AdviceSlipApi2 SDK exists test

require_once __DIR__ . '/../adviceslipapi2_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = AdviceSlipApi2SDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# AdviceSlipApi2 SDK exists test

require "minitest/autorun"
require_relative "../AdviceSlipApi2_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = AdviceSlipApi2SDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
